### DISCUSSION TITLE
How to to reset/retrieve password foor TOOLS menu

### DISCUSSION THREAD

**Discussion by NvBgm** [Nachtzuster/BirdNET-Pi#242] [Q&A] [answered]
I did a clean installation on raspberry and set my username and password. 
But i lost my password . Is there a way to reset or retrieve my password. ??
I can open raspberry by ssh

**Comment by alexbelgium** [comment, marked as answer]
Hi, do you mean for the rpi or birdnet-pi ?

For birdnet-pi interface, the username is always `birdnet` (hard coded) and the password is defined in the `CADDY_PWD` variable located in your configuration at `/etc/birdnet/birdnet.conf`

For the rpi itself, if you can login through ssh by using a saved session password, you could force to update your password with `sudo passwd <your_username>`

**Reply by NvBgm** [reply]
> Hi, do you mean for the rpi or birdnet-pi ?
> 
> For birdnet-pi interface, the username is always `birdnet` (hard coded) and the password is defined in the `CADDY_PWD` variable located in your configuration at `/etc/birdnet/birdnet.conf`
> 
> For the rpi itself, if you can login through ssh by using a saved session password, you could force to update your password with `sudo passwd <your_username>`

Thanks Only username Birdnet works for me
